fuse/poll_darwin.go

```diff
@@ -23,6 +23,9 @@ func sysPoll(fds []pollFd, timeout int) (n int, err error) {
 }
 
 func pollHack(mountPoint string) error {
+	if !osxFuse {
+		return nil
+	}
```

Comment:
Why doesn't fuse-t need the pollHack?

Reply:
when a mount function returns, it's assumed that the actual mount already finished a the mount point is available to the user. fuse-t actually takes care about that. But I can see you point and maybe this can hack can be left as as. I will test and re-submit.